doc/devdocs/settingsv2/xaml-island-tweaks.md

```diff
@@ -22,9 +22,9 @@ Interop.ShowWindow(coreWindowInterop.WindowHandle, Interop.SW_HIDE);
 ```
 3. Workaround to prevent XAML Island failing to render on Nvidia workstation graphics cards:
```

Comment:
I wonder if it is "XAML Island" or "XAML Islands"

Please DON'T CHANGE anything unless approved by a Core PT team member.

Reply:
would be "the XAML Island" as there is a single island host control we are using

Reply:
Ok

Reply:
@alannt777 , 
should I fix something?

Reply:
if you'd like, sure, else i can go in quickly